src/index.html

```diff
@@ -11,12 +11,53 @@
       content="ie=edge"
     />
     <title>Moyo header</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <a class="img-wraper">
+        <img
+          src="./images/logo.png"
+          alt="Logotype"
+        />
+      </a>
+      <nav>
+        <ul>
+          <li class="is-active">
+            <a href="#">Apple</a>
```

Comment:
Class names should represent the meaning of the content (not the styles or tag names). It's better to use more semantic class names, like `nav__item` and `nav__link`.